docs/devel/networking.md

```diff
@@ -192,8 +192,8 @@ stage, will look like:
 
 Once the VM is booted, libvirt will consume the interface xml definition and
 create a tap device - named after the `target` parameter. That tap device will
-have the in-pod bridge as its master, and the tap device's MAC address, and
-link MTU will be configured according to the values set in the domain xml.
+be attached to the in-pod bridge as its port, and the tap device's MAC address,
```

Comment:
`as its port` sounds weird, maybe just remove?

The original page makes it sounds like bridges have... primary and secondary ports. This is typically not the case for basic linux bridges, in which case we're good. However, if the original page is correct, then this version loses that distinction.